internal/migrations/changesets.go

```diff
@@ -22,6 +22,8 @@ const (
 var (
 	ErrNoMoreChunksToRead = errors.New("no more chunks to read")
 	ErrChangesetNotFound  = errors.New("changeset not found")
+
+	migrationCompleted = "migration completed. `migration` section of `genesis.json` and `migrate_from` config in `config.toml` is no longer relevant and should be removed"
```

Comment:
:+1: Just did as indicate and restarted the net network's node without trouble.  There's only one node, but I'm pretty sure we confirmed that the `migration` section of genesis.json has no impact on genesis app hash.